<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8"><script language="javascript" src="/module/jslib/jquery/jquery.js"></script>

<meta name='WebId' content='161'>

<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
<title>Hong Kong Polytechnic University Publications</title>
<meta name='Maketime' content='2023-04-13 19:42:47'>
<meta name="SiteName" content="Hong Kong Polytechnic University">
<meta name="SiteDomain" content="https://genomicmedicine.github.io/site/">
<meta name="SiteIDCode" content="">
<meta name="ColumnName" content="Publications">
<meta name="ColumnDescription" content="">
<meta name="ColumnKeywords" content="">
<meta name="ColumnType" content="">
<meta name="channel" content="Publications">
<meta name="chnlcatalog" content="">
<meta name="server" content="/site/col/col48303/index.html">
<meta name="i_webid" content="161">
<meta name="i_columnid" content="48303">
<meta name="i_articleid" content="0">
<meta name="pagetype" content="2">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<link rel="stylesheet" href="/site/images/6964/font-awesome.min.css">
<link href="/site/script/bootstrap.min.css" rel="stylesheet">
<link rel="stylesheet" type="text/css" href="/site/images/6964/encommon.css">
<!--[if lt IE 9]>
<script src="https://cdn.bootcss.com/html5shiv/3.7.3/html5shiv.js"></script>
<script src="https://cdn.bootcss.com/respond.js/1.4.2/respond.min.js"></script>
<script>
        $(function () {
            $("#old-ie").css("display", "block");
        });
    </script>

<![endif]-->
<script src="/site/script/jquery.min.js"></script>
<script src="/site/script/bootstrap.min.js"></script>

</head>
<body><div id='hiddenLocation' style='display:none;'><span id='ss_webid'>161</span><span id='col1_name'>Publications</span><span id='col1_url'>/col/col48303/index.html</span></div>
 <div id="old-ie">您的浏览器版本太低，请使用IE9（或以上）、谷歌、火狐等现代浏览器。360、QQ、搜狗等浏览器请使用极速模式。</div>
 <header class="headbar">
  <div class="container ">
    <div class="row">
      <div class="col-sm-12  col-md-12 col-xs-12 col-lg-12 ">
        <div class="logo">
          <a href="#"><img src="/site/images/6964/logo.png" class="img-responsive"></a>
        </div>
      </div>

    </div>
  </div>
</header>
<nav class="navbar navbar-default" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target=".navbar-collapse">
        <span class="sr-only"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
    </div>
    <div class="collapse navbar-collapse">
      <ul class="nav navbar-nav">
        <li class="dropdown"><a id="Home" href="/site/">Home</a></li>
        <li class="dropdown"><a id="Research" href="/site/col/col48302/index.html">Research</a></li>
        <li class="dropdown"><a id="Publications" href="/site/col/col48303/index.html">Publications</a></li>
        <li class="dropdown"><a id="Team" href="/site/col/col48304/index.html">Team</a></li>
        <li class="dropdown"><a id="Join us" href="/site/col/col48305/index.html">Join us</a> </li>
        <li class="dropdown"><a id="Contact" href="/site/col/col48306/index.html">Contact</a></li>
      </ul>
    </div>
  </div>
</nav>
<section class="banner_col">
  <img src="/site/images/6964/banner1.jpg" alt="" >

</section>
<section class="navigatorbg">
<div class="container">
  <div class="row">
    <div class="col-md-12">
      <div class="navigator">
        <table cellspacing="0"cellpadding="0"border="0"><tr><td><a href="/en"><span class="glyphicon glyphicon-home home"></span>Home</a></td><td>>><a href="/site/col/col48303/index.html"class="bt_link">Publications</a></td></tr></table>
<script language="javascript">
    $(document).ready(function(){
     $(".infocont").addClass('piclist');
 });
</script>

      </div>  
    </div>
  </div>
</div>
</section>
<section class="">
  <div class="container">
    <div class="row">
      <div class="col-md-12">
        <!-- 3 -->
        <div class="colbox2">
              <div class="column_right">
                 
                        <div class="column_tit">
                          <span class="glyphicon glyphicon-bookmark"></span>Publications

                        </div>
                  
              <div class="column_list">
                <div class="infolist" style="padding-top: 3rem;">
                  <div class="artmain">
	<div class="infocont">
	<h3 class="art_t"><!--<$[标题]>begin-->Publications<!--<$[标题]>end--></h3>
	<div class="art_time">
	</div>
		<!--<$[信息内容]>begin--><!--ZJEG_RSS.content.begin--><meta name="ContentStart"><p style="font-size: 16px; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; text-align: left;"><a></a><strong><span style="font-size: 21px;font-family: Arial, sans-serif;">REPRESENTATIVE PUBLICATIO</span></strong><strong><span style="font-size: 21px;font-family: Arial, sans-serif;">NS</span><span style="font-family: Arial, sans-serif; font-size: 21px;">&nbsp;from Prof. Zhang Weixiong&nbsp;</span></strong><span style="font-size: 15px;font-family: Arial, sans-serif;">(TOTAL&gt;200)</span></p><p style="margin: 0px 0px 0px 48px; font-size: 16px; font-family: &quot;CMB X 12&quot;; white-space: normal; text-align: left; line-height: normal;"><span style="font-size: 15px;font-family: Symbol;color: windowtext;">·<span style="font-variant-numeric: normal;font-variant-east-asian: normal;font-variant-alternates: normal;font-kerning: auto;font-optical-sizing: auto;font-feature-settings: normal;font-variation-settings: normal;font-stretch: normal;font-size: 9px;line-height: normal;font-family: 'Times New Roman';">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span style="font-size: 15px;font-family: Arial, sans-serif;">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(196, 89, 17);">genom</span><span style="font-size: 15px;font-family: Arial, sans-serif;">] – </span><em><span style="font-size: 15px; font-family: Arial, sans-serif; color: windowtext; text-decoration: underline;">Genomics </span></em><span style="font-size: 15px; font-family: Arial, sans-serif; color: windowtext; text-decoration: underline;">and</span><em><span style="font-size: 15px; font-family: Arial, sans-serif; color: windowtext; text-decoration: underline;"> transcriptomics</span></em><span style="font-size: 15px;font-family: Arial, sans-serif;color: windowtext;">,</span></p><p style="margin: 0px 0px 0px 48px; font-size: 16px; font-family: &quot;CMB X 12&quot;; white-space: normal; text-align: left; line-height: normal;"><span style="font-size: 15px;font-family: Symbol;color: windowtext;">·<span style="font-variant-numeric: normal;font-variant-east-asian: normal;font-variant-alternates: normal;font-kerning: auto;font-optical-sizing: auto;font-feature-settings: normal;font-variation-settings: normal;font-stretch: normal;font-size: 9px;line-height: normal;font-family: 'Times New Roman';">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span style="font-size: 15px;font-family: Arial, sans-serif;">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(46, 116, 181);">ncRNA</span><span style="font-size: 15px;font-family: Arial, sans-serif;">] – </span><em><span style="font-size: 15px; font-family: Arial, sans-serif; color: windowtext; text-decoration: underline;">Noncoding RNA</span></em><span style="font-size: 15px;font-family: Arial, sans-serif;color: windowtext;">,</span></p><p style="margin: 0px 0px 0px 48px; font-size: 16px; font-family: &quot;CMB X 12&quot;; white-space: normal; text-align: left; line-height: normal;"><span style="font-size: 15px;font-family: Symbol;color: windowtext;">·<span style="font-variant-numeric: normal;font-variant-east-asian: normal;font-variant-alternates: normal;font-kerning: auto;font-optical-sizing: auto;font-feature-settings: normal;font-variation-settings: normal;font-stretch: normal;font-size: 9px;line-height: normal;font-family: 'Times New Roman';">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span style="font-size: 15px;font-family: Arial, sans-serif;">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(191, 143, 0);">medImg</span><span style="font-size: 15px;font-family: Arial, sans-serif;">] – </span><em><span style="font-size: 15px; font-family: Arial, sans-serif; color: windowtext; text-decoration: underline;">Medical imaging</span></em><span style="font-size: 15px;font-family: Arial, sans-serif;color: windowtext;">,</span></p><p style="margin: 0px 0px 0px 48px; font-size: 16px; font-family: &quot;CMB X 12&quot;; white-space: normal; text-align: left; line-height: normal;"><span style="font-size: 15px;font-family: Symbol;color: windowtext;">·<span style="font-variant-numeric: normal;font-variant-east-asian: normal;font-variant-alternates: normal;font-kerning: auto;font-optical-sizing: auto;font-feature-settings: normal;font-variation-settings: normal;font-stretch: normal;font-size: 9px;line-height: normal;font-family: 'Times New Roman';">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span style="font-size: 15px;font-family: Arial, sans-serif;">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(68, 114, 196);">ml/dm</span><span style="font-size: 15px;font-family: Arial, sans-serif;">] – </span><em><span style="font-size: 15px; font-family: Arial, sans-serif; color: windowtext; text-decoration: underline;">Machine learning</span></em><span style="font-size: 15px;font-family: Arial, sans-serif;color: windowtext;"> and </span><em><span style="font-size: 15px; font-family: Arial, sans-serif; color: windowtext; text-decoration: underline;">big data analytics</span></em><span style="font-size: 15px;font-family: Arial, sans-serif;color: windowtext;">, and</span></p><p style="margin: 0px 0px 8px 48px; font-size: 16px; font-family: &quot;CMB X 12&quot;; white-space: normal; text-align: left; line-height: normal;"><span style="font-size: 15px;font-family: Symbol;color: windowtext;">·<span style="font-variant-numeric: normal;font-variant-east-asian: normal;font-variant-alternates: normal;font-kerning: auto;font-optical-sizing: auto;font-feature-settings: normal;font-variation-settings: normal;font-stretch: normal;font-size: 9px;line-height: normal;font-family: 'Times New Roman';">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span style="font-family: Arial, sans-serif;">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(244, 176, 131);">AI</span><span style="font-family: Arial, sans-serif;">] – </span><em><span style="font-size: 15px; font-family: Arial, sans-serif; color: windowtext; text-decoration: underline;">Artificial intelligence</span></em><span style="font-size: 15px;font-family: Arial, sans-serif;color: windowtext;"> (heuristic search, constraint satisfaction, and optimization)</span></p><p style="margin: 8px 0px; font-size: 16px; font-family: &quot;CMB X 12&quot;; white-space: normal; text-align: left; line-height: normal;"><strong><em><span style="font-size: 19px;font-family: Arial, sans-serif;color: windowtext;"><br></span></em></strong></p><p style="margin: 8px 0px; font-size: 16px; font-family: &quot;CMB X 12&quot;; white-space: normal; text-align: left; line-height: normal;"><strong><em><span style="font-size: 19px;font-family: Arial, sans-serif;color: windowtext;">Research Monograph</span></em></strong></p><p style="margin: 8px 0px; font-size: 16px; font-family: &quot;CMB X 12&quot;; white-space: normal; text-align: left; line-height: normal;"><span style="font-size: 15px; font-family: Symbol; color: windowtext;">·<span style="font-variant-numeric: normal;font-variant-east-asian: normal;font-variant-alternates: normal;font-kerning: auto;font-optical-sizing: auto;font-feature-settings: normal;font-variation-settings: normal;font-stretch: normal;font-size: 9px;line-height: normal;font-family: 'Times New Roman';">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span style="font-family: Arial, sans-serif;">[</span><span style="font-size: 15px; font-family: Arial, sans-serif; color: rgb(244, 176, 131);">AI</span><span style="font-family: Arial, sans-serif;">] </span><strong><span style="font-size: 15px;font-family: Arial, sans-serif;color: windowtext;">W. Zhang</span></strong><span style="font-size: 15px; font-family: Arial, sans-serif; color: windowtext;">, <em>State-Space Search: Algorithms, Complexity, Extensions, and Applications</em>, Springer-Verlag, New York, NY, 1999.</span></p><p style="margin: 8px 0px; font-size: 16px; font-family: &quot;CMB X 12&quot;; white-space: normal; text-align: left; line-height: normal;"><strong style="font-family: Calibri, sans-serif; font-size: 15px;"><em><span style="font-size: 19px;font-family: Arial, sans-serif;"><br></span></em></strong></p><p style="margin: 8px 0px; font-size: 16px; font-family: &quot;CMB X 12&quot;; white-space: normal; text-align: left; line-height: normal;"><strong style="font-family: Calibri, sans-serif; font-size: 15px;"><em><span style="font-size: 19px;font-family: Arial, sans-serif;">Selected Journal Publication</span></em></strong></p><p style="margin: 0px 0px 8px 24px; font-size: 16px; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; text-align: left;"><a style="text-decoration: underline; font-family: Arial, sans-serif; font-size: 14px;"><span style="font-family: Arial, sans-serif; font-size: 14px;"></span></a></p><ol style="list-style-type: decimal;" class=" list-paddingleft-2"><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><a></a><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[<span style="color: rgb(196, 89, 17); font-family: Arial, sans-serif; font-size: 15px; text-wrap: wrap;">genom</span></span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] X. Zhang, C. Pan, X. Wei, M. Yu, S. Liu, J. An, J. Yang, B. Wei, W. Hao, Y. Yao, Y. Zhu,&nbsp;<span style="font-family: Arial, sans-serif; font-size: 15px; text-wrap: wrap; background-color: rgb(255, 255, 255);">and&nbsp;</span><strong style="font-family: Arial, sans-serif; font-size: 15px; text-wrap: wrap; background-color: rgb(255, 255, 255);">W. Zhang<span style="font-family: Arial, sans-serif; font-size: 15px; text-wrap: wrap; background-color: rgb(255, 255, 255);"></span></strong><span style="font-family: Arial, sans-serif; font-size: 15px; text-wrap: wrap; background-color: rgb(255, 255, 255);">, accepted to </span></span><span style="text-wrap: wrap; background-color: rgb(255, 255, 255); font-size: 15px; font-family: Arial, sans-serif; color: rgb(51, 51, 51); text-decoration: underline;"><em>iScience</em></span><span style="text-wrap: wrap; background-color: rgb(255, 255, 255); font-size: 15px; font-family: Arial, sans-serif; color: rgb(51, 51, 51);"><span style="text-wrap: wrap; background-color: rgb(255, 255, 255); font-size: 15px; font-family: Arial, sans-serif; color: rgb(51, 51, 51); text-decoration: none;">.</span></span></p></li><li><p style="margin-top: 8px; margin-bottom: 8px; font-family: Calibri, sans-serif; background: white; text-align: left;"><span style="font-size: 15px; font-family: Arial, sans-serif; color: rgb(51, 51, 51);">[</span><span style="font-size: 15px; font-family: Arial, sans-serif; color: rgb(68, 114, 196);">ml/dm</span><span style="font-size: 15px; font-family: Arial, sans-serif; color: rgb(51, 51, 51);">]&nbsp;</span><span style="font-size: 15px; font-family: Arial, sans-serif;">D. He, C. Liang, C. Huo, Z. Feng, D. Jin, L. Yang, and&nbsp;<strong>W. Zhang</strong>, Analyzing heterogeneous networks with missing attributes by unsupervised contrastive learning,&nbsp;<em><span style="text-decoration-line: underline;">IEEE Trans. On Neural Networks and Learning Systems</span></em>, in press,&nbsp;published online March 2022.&nbsp;</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">&nbsp;[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(68, 114, 196);">ml/dm</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">]&nbsp;D. Jin, Z. Yu, P. Jiao, S. Pan, D. He, J. Wu, P. Yu, and&nbsp;<strong>W. Zhang</strong>,&nbsp;A survey of community detection approaches: From statistical modeling to deep learning,&nbsp;<em><span style="text-decoration:underline;">IEEE Trans. on Knowledge and Data Engineering</span></em>, 2023; 35(2):1149-90.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(46, 116, 181);">ncRNA</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">]&nbsp;</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: black;">P. Xuan</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">*,&nbsp;</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: black;">L. Zhan</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">,&nbsp;</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: black;">H. Cui</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">,&nbsp;</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: black;">T. Zhang</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">,&nbsp;</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: black;">T. Nakaguchi</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">,&nbsp;</span><strong><span style="font-size: 15px;font-family: Arial, sans-serif;color: black;">W. Zhang</span></strong><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">, Graph Triple-Attention Network for Disease-Related LncRNA Prediction,&nbsp;<em><span style="text-decoration:underline;">IEEE J Biomedical and Health Informatics</span></em>, 2022 Jun;26(6):2839-49.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(196, 89, 17);">genom</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] M. Chang, F. Womer, X. Gong, X. Chen, L. Tang, R. Feng, S. Dong, J. Duan, Y. Chen, R. Zhang, Y. Wang, S. Ren, Y. Wang, J. Kang, Z. Yin, Y. Wei, S. Wei, X. Jian, K. Xu, B. Cao, Y. Zhang,&nbsp;<strong>W. Zhang</strong>, Y. Tang, X. Zhang, F. Wang*,&nbsp;Identifying and validating subtypes within major psychiatric disorders based on frontal-posterior functional imbalance via deep learning,&nbsp;<em><span style="text-decoration:underline;">Molecular Psychiatry</span></em>, 26(7):2991-3002, 2021.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(196, 89, 17);">genom</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] L. Chen, J. Zhou, T. Li, Z. Fang, L. Li, G. Huang, L. Gao, X. Zhu, X. Zhou, H. Xiao, J. Zhang, Q. Xiong, J. Zhang, A. Ma, W. Zhai,&nbsp;<strong>W. Zhang</strong>*, and H. Peng*,&nbsp;GmoDetector: An accurate and efficient GMO identification approach and its application,&nbsp;<em><span style="text-decoration:underline;">Food Research Intern.</span></em>, 149:110662, 2021.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(46, 116, 181);">ncRNA</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] X. Liu, J. Frost, A. Bowcock,&nbsp;<strong>W. Zhang</strong>*, Canonical and interior circular RNAs function as competing endogenous RNAs in psoriatic skin,&nbsp;<em><span style="text-decoration:underline;">Intern. J. Molecular Sciences</span></em>, 22(10), 5182, 2021.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(191, 143, 0);">medImg</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] Z. Zhang, T. Zhao, H. Gay,&nbsp;<strong>W. Zhang*</strong>, B. Sun*,&nbsp;Weaving attention U-net: A novel hybrid CNN and attention-based method for organs-at-risk segmentation in head and neck CT images,&nbsp;<em><span style="text-decoration:underline;">Medical Physics</span></em>, 48(11):7052-62, 2021.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(191, 143, 0);">medImg</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] Z. Zhang, T. Zhao, H. Gay,&nbsp;<strong>W. Zhang*</strong>, B. Sun*,&nbsp;ARPM-net: A novel CNN-based adversarial method with Markov Random Field enhancement for prostate and organs at risk segmentation in pelvic CT images,&nbsp;<em><span style="text-decoration:underline;">Medical Physics</span></em>, 48(1):227-37, 2021.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(191, 143, 0);">medImg</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] Z. Zhang, T. Zhao, H. Gay,&nbsp;<strong>W. Zhang*</strong>, B. Sun*,&nbsp;Semi-supervised semantic segmentation of prostate and organs-at-risk on 3D pelvic CT images&nbsp;<em><span style="text-decoration:underline;">Biomedical Physics Engineering Express</span></em>.&nbsp;7(6), 2021</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(68, 114, 196);">ml/dm</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">]&nbsp;D. Jin, X. Wang, D. He, J. Dang,&nbsp;<strong>W. Zhang</strong>,&nbsp;Robust detection of link communities with summary description in social networks,&nbsp;<em><span style="text-decoration:underline;">IEEE Transactions on Knowledge and Data Engineering</span></em>, 33(6):2737-49, 2021.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(196, 89, 17);">genom</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] L. Li, H. Peng, S. Tan, J. Zhou, Z. Fang, Z. Hu, L. Gao, T. Li,&nbsp;<strong>W. Zhang</strong>*, and L. Chen*,&nbsp;Effects of early cold stress on gene expression in Chlamydomonas reinhardtii,&nbsp;<em><span style="text-decoration:underline;">Genomics</span></em>, 112(2):1128-38, 2020.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(46, 116, 181);">ncRNA</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] X. Liu, Z. Hu, J. Zhou, C. Tian, G. Tian, M. He, L. Gao, L. Chen, T. Li, P. Peng,&nbsp;<strong>W. Zhang*</strong>, Interior circular RNA,&nbsp;<em><span style="text-decoration:underline;">RNA Biology</span></em>, 17(1):87-97, 2020.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(68, 114, 196);">ml/dm</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] M. Li,&nbsp;D. Jin, D. He, and&nbsp;<strong>W. Zhang</strong>,&nbsp;Modeling with node popularities for autonomous overlapping community detection,&nbsp;<em><span style="text-decoration:underline;">ACM Transactions on Information Systems</span></em>, 11(3):27, 2020.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(191, 143, 0);">medImg</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] D. Lam, X. Zhang, H. Li, Y. Deshan, B. Schott, T. Zhao,&nbsp;<strong>W. Zhang</strong>, S. Mutic, and B. Sun*,&nbsp;Predicting gamma passing rates for portal dosimetry-based IMRT QA using machine learning,&nbsp;<em><span style="text-decoration:underline;">Medical Physics</span></em>, 46(10):4666-75, 2019.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(68, 114, 196);">ml/dm</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] L. Yang, Y. Wang, J. Gu, X. Cao, X. Wang, D. Jin, G. Ding, J. Han, and&nbsp;<strong>W. Zhang</strong>, Autonomous semantic community detection via adaptively weighted low-rank approximation,&nbsp;<em><span style="text-decoration:underline;">ACM Transactions on Multimedia Computing</span></em><span style="text-decoration:underline;">,&nbsp;<em>Communications, and Applications</em></span>&nbsp;(TOMM), 15(3s), 2019.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(196, 89, 17);">genom</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] L. Li, G. Tian, H. Peng, D. Meng, L. Wang, X. Hu, C. Tian, M. He, J. Zhou, L. Chen, C. Fu,&nbsp;<strong>W. Zhang</strong>* and Z. Hu*, New class of transcription factors controls flagellar assembly by recruiting RNA polymerase II in&nbsp;<em>Chlamydomonas</em>,&nbsp;<em><span style="text-decoration:underline;">Proc of National Academy of Sciences of the USA</span>&nbsp;</em>(<em><span style="text-decoration:underline;">PNAS</span></em>), 115(17):4435-4440, 2018.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(46, 116, 181);">ncRNA</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] Y. Zhong, P. Xuan*, X. Wang, T. Zhang, J. Li, Y. Liu and&nbsp;<strong>W. Zhang</strong>,&nbsp;A non-negative matrix factorization based method for predicting disease-associated miRNAs in miRNA-disease bilayer network,&nbsp;<em><span style="text-decoration:underline;">Bioinformatics</span></em>, 34(2):267-77, 2018.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(196, 89, 17);">genom</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] L. Li, Z. Fang, J. Zhou, H. Chen, Z. Hu, L. Gao, L. Chen, S. Ren, H. Ma, L. Lu,&nbsp;<strong>W. Zhang</strong>* and H. Peng*, An accurate and efficient method for large-scale SSR genotyping and applications,&nbsp;<em><span style="text-decoration:underline;">Nucleic Acids Research</span></em>, 45(10):e88, 2017.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(196, 89, 17);">genom</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] T.P. Michael, D. Bryant, R. Gutierrez, N. Borisjuk, P. Chu, H. Zhang, J. Xia, J. Zhou, H. Peng, M.E. Baidouri, B. ten Hallers, AR. Hastie, T. Liang, K. Acosta, S. Gilbert, C. McEntee, SA. Jackson, TC. Mockler,&nbsp;<strong>W. Zhang</strong>&nbsp;and E. Lam*,&nbsp;Comprehensive definition of genome features in&nbsp;<em>Spirodela polyrhiza</em>&nbsp;by high-depth physical mapping and short-read DNA sequencing strategies,&nbsp;<em><span style="text-decoration:underline;">The Plant Journal</span></em>,&nbsp;89(3):617-35, 2017.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(46, 116, 181);">ncRNA</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] J. Xia, L. Li, T. Li, Z. Fang, K. Zhang, J. Zhou, H. Peng, and&nbsp;<strong>W. Zhang</strong>*, Detecting and characterizing microRNAs of diverse genomic origins via miRvial,&nbsp;<em><span style="text-decoration:underline;">Nucleic Acids Research</span></em>, 45(21):e176, 2017.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(68, 114, 196);">ml/dm</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] X. Zhang*, J. Han and W.&nbsp;<strong>Zhang</strong>,&nbsp;An efficient algorithm for finding all possible input nodes for controlling complex networks,<em><span style="text-decoration:underline;">&nbsp;Scientific Reports</span></em>, 7(1):10677, 2017.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(68, 114, 196);">ml/dm</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] J. Fu,&nbsp;<strong>W. Zhang</strong>&nbsp;and J. Wu*,&nbsp;Identification of leader and self-organizing communities in complex networks,&nbsp;<em><span style="text-decoration:underline;">Scientific Reports</span></em>, 7(1):704, 2017.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(196, 89, 17);">genom</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] D. Tiosano, L. Audi, S. Climer,&nbsp;<strong>W. Zhang</strong>, A.R. Templeton*, M. Fernandez-Cancio, R. Gershoni-Baruch, J.M. Sanchez-Muro, M.E. Kholy, Z. Hochberg,&nbsp;Latitudinal Clines of the human Vitamin D receptor and skin&nbsp;color-genes,&nbsp;<em><span style="text-decoration:underline;">Genes, Genomes, Genetics</span></em>, 6(5):1251-66, 2016.&nbsp;(Based on the work receiving</span><strong><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);"> the&nbsp;</span><em><span style="text-decoration:underline;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(91, 155, 213);">Henning Anderson Price</span></span></em></strong><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">&nbsp;at the European Society for Paediatric Endocrinology 51<sup>st</sup>&nbsp;Annual Meeting)</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(196, 89, 17);">genom</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] S. Climer*, A. Templeton and&nbsp;<strong>W. Zhang</strong>*,&nbsp;Human&nbsp;<em>gephyrin</em>&nbsp;is encompassed within giant functional noncoding yin-yang sequences,&nbsp;<em><span style="text-decoration:underline;">Nature Communications</span></em>, 6:6534, 2015.&nbsp;</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(46, 116, 181);">ncRNA</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] D. Nie, J. Xia, C. Jiang, B. Qi, X. Ling, S. Lin,&nbsp;<strong>W. Zhang</strong>, J. Guo, H. Jin and H. Zhao,&nbsp;Bacillus cereus AR156 primes induced systemic resistance by suppressing miR825/825* and activating defense related genes in Arabidopsis,&nbsp;<em><span style="text-decoration:underline;">J. Integrative Plant Biology</span></em>, 58(4):426-39, 2015.&nbsp;</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(68, 114, 196);">ml/dm</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">]&nbsp;D. He, D. Jin, Z. Chen and&nbsp;<strong>W. Zhang*</strong>,&nbsp;Identification of hybrid node and link communities in complex networks,&nbsp;<em><span style="text-decoration:underline;">Scientific Reports</span></em>, 5:8638, 2015.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(196, 89, 17);">genom</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">]&nbsp;W. Wang, B. Feng, J. Xiao, Z. Xia, X. Zhou, P. Li,&nbsp;<strong>W. Zhang</strong>, et al.,&nbsp;Cassava genome from a wild ancestor to cultivated varieties<em>,&nbsp;<span style="text-decoration:underline;">Nature Communications</span></em>, 5:5110, 2014.&nbsp;</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(196, 89, 17);">genom</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">]&nbsp;S. Climer, A. Templeton and&nbsp;<strong>W. Zhang</strong>*,&nbsp;Allele-specific network reveals combinatorial interactions that transcends small effects in psoriasis GWAS,&nbsp;<em><span style="text-decoration:underline;">PLOS Computational Biology</span></em>, 10(9):e1003766, 2014.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(46, 116, 181);">ncRNA</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] J. Xia and&nbsp;<strong>W. Zhang*,&nbsp;</strong>MicroRNAs in normal and psoriatic skin,&nbsp;<em><span style="text-decoration:underline;">Physiological Genomics</span></em>, 46(4):113-22, 2014,&nbsp;<span style="text-decoration:underline;">invited review</span>.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(46, 116, 181);">ncRNA</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] J. Xia and&nbsp;<strong>W. Zhang*</strong>, A meta-analysis revealed insights into the sources, conservation and impact of microRNA 5'-isoforms in four model species,&nbsp;<em><span style="text-decoration:underline;">Nucleic Acids Research</span></em>, 42(3):1427-41, 2014.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(196, 89, 17);">genom</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] Z. Chen and&nbsp;<strong>W. Zhang</strong>*, Integrative analysis using module-guided Random Forests reveals correlated genetic factors related to mouse weight,&nbsp;<em><span style="text-decoration:underline;">PLOS Computational Biology</span></em>, 9(3):e1002956, 2013.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(46, 116, 181);">ncRNA</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] J. Xia, C.E. Joyce, A.M. Bowcock* and&nbsp;<strong>W. Zhang</strong>*, Noncanonical microRNAs and endogenous siRNAs in normal and psoriatic human skin,&nbsp;<em><span style="text-decoration:underline;">Human Molecular Genetics</span></em>, 22(4):737-48, 2013.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(244, 176, 131);">AI</span><span style="font-size: 15px;font-family: Arial, sans-serif;">] Q. Lu, R. Huang, Y. Chen, Y. Xu,&nbsp;<strong>W. Zhang</strong>, G. Sun and G. Chen, A SAT-based approach to cost sensitive temporally expressive planning,&nbsp;<em><span style="text-decoration:underline;">ACM Transactions on Intelligent Systems and Technology</span></em>, 5(1):18, 2013.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(46, 116, 181);">ncRNA</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] X. Zhang, X. Jin, Y. Lii, B.E. Barrera-Figueroa, X. Zhou, S. Gao, L. Lu, D. Nie, Z. Chen, C. Leung, T. Wong, H. Zhang, J. Guo, Y. Li, R. Liu, W. Liang, J-K. Zhu,&nbsp;<strong>W. Zhang</strong>*, H. Jin*, Genome-wide analysis of plant nat-siRNAs reveals insights into their distribution, biogenesis and function,&nbsp;<em><span style="text-decoration:underline;">Genome Biology</span></em>, 13:R20, 2012.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(46, 116, 181);">ncRNA</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] Y. Zheng*, Y-F. Li, R. Sunkar and&nbsp;<strong>W. Zhang</strong>, SeqTar: An effective method for identifying microRNA guided cleavage sites from degradome of polyadenylated transcripts in plants,&nbsp;<em><span style="text-decoration:underline;">Nucleic Acids Research</span></em>, 40(4):e28, 2012.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(244, 176, 131);">AI</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] R. Huang, Y. Chen and&nbsp;<strong>W. Zhang</strong>&nbsp;(2012) SAS+ planning as Satisfiability,<em>&nbsp;<span style="text-decoration:underline;">J. Artificial Intelligence Research</span>,&nbsp;</em>43:293-328, 2012 (</span><strong><span style="text-decoration:underline;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(91, 155, 213);">the AAAI-2010&nbsp;<em>Outstanding Paper Award</em></span></span></strong><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">).</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(91, 155, 213);">ncRNA</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] C.E. Joyce, X. Zhou, J. Xia, C. Ryan, B. Thrash, A. Menter,&nbsp;<strong>W. Zhang</strong>* and A.M. Bowcock*, Deep sequencing of small RNAs from human skin reveals major&nbsp;alterations in the psoriasis miRNAome,&nbsp;<em>Human Molecular Genetics</em>,&nbsp;&nbsp;20(20):4025-&nbsp; 40, 2011.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(91, 155, 213);">ncRNA</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">]&nbsp;<strong>W. Zhang</strong>*, S. Guo, J. Xia, X. Zhou, P. Chellappan, X. Zhou, X. Zhang and H. Jin*, Multiple distinct small RNAs originated from the same microRNA&nbsp;precursors,&nbsp;<em>Genome Biology</em>, 11:R81, 2010.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(91, 155, 213);">ncRNA</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] P. Chellappan, J. Xia, X. Zhou, S. Gao, X. Zhang, G. Coutino, F. Vazquez,&nbsp;<strong>W. Zhang</strong>* and H. Jin*, siRNAs from miRNA sites mediate DNA methylation of&nbsp;target genes,&nbsp;<em>Nucleic Acids Research</em>, 38(20):6883-94, 2010.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(91, 155, 213);">ncRNA</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] C. Zeng, W. Wang, Y. Zheng, X. Chen, X. Bo, S. Song,&nbsp;<strong>W. Zhang</strong>*, M. Peng*, Conservation and divergence of microRNAs and their functions in&nbsp;Euphorbiaceous plants,&nbsp;<em>Nucleic Acids Research</em>, 38(3):981-&nbsp; 95, 2010.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(91, 155, 213);">ncRNA</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] T.A. Reese, J. Xia, L.S. Johnson, X. Zhou,&nbsp;<strong>W. Zhang</strong>* and H.W. Virgin*, Identification of novel microRNA-like molecules generated from herpesvirus and host&nbsp;tRNA transcripts,&nbsp;<em>J. Virology</em>, 84(19):10344-53, 2010.<br>[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(91, 155, 213);">ncRNA</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] Y-F. Li, Y. Zheng, C. Addo-Quaye, L. Zhang, A. Saini, G. Jagadeeswaran, M. Axtell,&nbsp;<strong>W. Zhang</strong>, R. Sunkar*, Transcriptome-wide identification of microRNA&nbsp;targets in rice,&nbsp;<em><span style="text-decoration:underline;">The Plant Journal</span></em>, 62(5):742-59, 2010.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(244, 176, 131);">AI</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] G. Jaeger and&nbsp;<strong>W. Zhang&nbsp;</strong>(2010) An efficient algorithm for and phase transitions of the directed Hamiltonian cycle problem,<em>&nbsp;<span style="text-decoration:underline;">J. Artificial Intelligence Research</span>,&nbsp;</em>39:663-87, 2010.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(196, 89, 17);">genom</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] J.A. Webster, J.R. Cibbs, J. Clarke, M. Ray,&nbsp;<strong>W. Zhang</strong>, P. Holmans, K. Rohrer, A. Zhao, L. Marlowe, M. Kaleem, D.S. McCorquodale III, C. Cuello, D. Leung, L. Bryden, P. Nath, V.L. Zisman, K. Joshipura, M.J. Huentelman, D. H Lince, K.D. Coon, D.W. Craig, J.V. Pearson, C.B. Heward, E.M. Reiman, D. Stephan, J. Hardy, A.J. Myers*, Genetic control of human brain transcript expression in Alzheimer's disease,&nbsp;<em><span style="text-decoration:underline;">American J. of Human Genetics</span></em>, 84:445-58, 2009.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(196, 89, 17);">genom</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] S. Climer, G. Jaegerg, A. Templeton and&nbsp;<strong>W. Zhang*</strong>, How frugal is mother nature with haplotypes?&nbsp;<em><span style="text-decoration:underline;">Bioinformatics</span></em>, 25(1):68-74, 2009.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(91, 155, 213);">ncRNA</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] X. Zhou, R. Sunkar, H. Jin, J-K. Zhu and&nbsp;<strong>W. Zhang</strong>*, Genome-wide identification and analysis of small RNAs originated from natural antisense transcripts in Oryza sativa,&nbsp;<em><span style="text-decoration:underline;">Genome Research</span></em>, 19:70-8, 2009.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(91, 155, 213);">ncRNA</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] G. Jagadeeswaran, Y. Zheng, Y-f. Li, L.I. Shukla, J. Matts, P. Hoyt, S.L. Macmil, G.B. Wiley, B.A. Roe,&nbsp;<strong>W. Zhang</strong>, R. Sunkar*, Cloning and characterization of small RNAs from Medicago truncatula reveals novel legume-specific and candidate microRNAs,&nbsp;<em><span style="text-decoration:underline;">New Phytologist</span></em>, 184(1):85-98, 2009.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(244, 176, 131);">AI</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] Y. Chen, R. Huang, Z. Xing and&nbsp;<strong>W. Zhang</strong>&nbsp;(2009) Long-distance mutual exclusion for planning,&nbsp;<em><span style="text-decoration:underline;">Artiﬁcial Intelligence</span></em>, 173:365-91, 2009.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(196, 89, 17);">genom</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] M. Ray J. Ruan and W. Zhang*, Variations in the transcriptome of Alzheimer's disease reveal molecular networks involved in cardiovascular diseases,&nbsp;<em><span style="text-decoration:underline;">Genome Biology</span></em>, 9(10):R148, 2008.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(46, 116, 181);">ncRNA</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">]&nbsp;</span><span style="font-size: 15px;font-family: Arial, sans-serif;">Y. Xu, X. Zhou and&nbsp;<strong>W. Zhang*</strong>, MicroRNA prediction with a novel ranking algorithm based on random walks,&nbsp;<em><span style="text-decoration:underline;">Bioinformatics</span></em>, 24:i50-8, 2008.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(196, 89, 17);">genom</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">]&nbsp;</span><span style="font-size: 15px;font-family: Arial, sans-serif;">M. Ray, S. Dharmarajan, J. Freudenberg,&nbsp;<strong>W. Zhang*</strong>&nbsp;and G.A. Patterson*, Use of gene expression proﬁling and machine learning to understand primary graft dysfunction,&nbsp;<em><span style="text-decoration:underline;">American J. of Transplantation</span></em>, 7:2396-405, 2007.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(196, 89, 17);">genom</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">]&nbsp;</span><span style="font-size: 15px;font-family: Arial, sans-serif;">X. Zhou, J. Ruan, G. Wang and&nbsp;<strong>W. Zhang*</strong>, Characterization and identification of microRNA core promoters in four model species,&nbsp;<em><span style="text-decoration:underline;">PLoS Computational Biology</span></em>, 3(3):e37, 2007.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(196, 89, 17);">genom</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">]&nbsp;</span><span style="font-size: 15px;font-family: Arial, sans-serif;">H. Zeng, L. Luo,&nbsp;<strong>W. Zhang</strong>, J. Zhou, Z. Li, H. Liu, T. Zhu, X. Feng and Y. Zhong*, PlantQTL-GE: A database for searching candidate genes by gene expression and QTL information,&nbsp;<em><span style="text-decoration:underline;">Nucleic Acids Research</span></em>, 35:D879-D882, 2007.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(46, 116, 181);">ncRNA</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">]&nbsp;</span><span style="font-size: 15px;font-family: Arial, sans-serif;">X. Zhou, G. Wang and&nbsp;<strong>W. Zhang*</strong>, UV-B light responsive microRNA genes in Arabidopsis thaliana,&nbsp;<em><span style="text-decoration:underline;">Molecular Systems Biology</span></em>, 3:103, 2007.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(196, 89, 17);">genom</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">]&nbsp;</span><span style="font-size: 15px;font-family: Arial, sans-serif;">G. Wang and W.&nbsp;<strong>Zhang*</strong>, A steganalysis-based approach to comprehensive identiﬁcation and characterization of functional regulatory elements,&nbsp;<em><span style="text-decoration:underline;">Genome Biology</span></em>, 7(6):R49, 2006.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(196, 89, 17);">genom</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">]&nbsp;</span><span style="font-size: 15px;font-family: Arial, sans-serif;">J.Ruan and&nbsp;<strong>W. Zhang*</strong>, A two-dimensional regression tree approach to the modeling of gene expression regulations,&nbsp;<em><span style="text-decoration:underline;">Bioinformatics</span></em>, 22(3):332-40, 2006.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(68, 114, 196);">ml/dm</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">]&nbsp;</span><span style="font-size: 15px;font-family: Arial, sans-serif;">S. Climer and&nbsp;<strong>W. Zhang</strong>, Rearrangement clustering: Pitfalls, remedies and applications,&nbsp;<em><span style="text-decoration:underline;">J. Machine Learning Research</span></em>, 7:919-43, 2006.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(244, 176, 131);">AI</span><span style="font-size: 15px;font-family: Arial, sans-serif;">] S. Climer and&nbsp;<strong>W. Zhang</strong>&nbsp;(2006) Cut-and-solve: A linear search strategy for combinatorial optimization problems,&nbsp;<em><span style="text-decoration:underline;">Artiﬁcial Intelligence</span></em>, 170(8-9):714-38, 2006.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(196, 89, 17);">genom</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">]&nbsp;</span><strong><span style="font-size: 15px;font-family: Arial, sans-serif;">W. Zhang*</span></strong><span style="font-size: 15px;font-family: Arial, sans-serif;">, J.Ruan, T-h. D.Ho, Y. You, T. Yu and R.S. Quatrano, Cis-regulatory element based targeted gene ﬁnding: Genome-wide identiﬁcation of ABA and abiotic stress responsive genes in Arabidopsis thaliana,&nbsp;<em><span style="text-decoration:underline;">Bioinformatics</span></em>, 21(14):3074-81, 2005.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(196, 89, 17);">genom</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">]&nbsp;</span><span style="font-size: 15px;font-family: Arial, sans-serif;">G. Wang, T. Yu and W.&nbsp;<strong>Zhang*</strong>, WordSpy: Identify transcription factor biding motifsby building a dictionary and learning a grammar,&nbsp;<em><span style="text-decoration:underline;">Nucleic Acids Research</span></em>, 33:W4126, 2005.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(244, 176, 131);">AI</span><span style="font-size: 15px;font-family: Arial, sans-serif;">] R. Korf,&nbsp;<strong>W. Zhang</strong>, I. Thayer and H. Hohwald (2005) Frontier search,&nbsp;<em><span style="text-decoration:underline;">J. ACM</span></em>, 52(5):71548, 2005.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(244, 176, 131);">AI</span><span style="font-size: 15px;font-family: Arial, sans-serif;">] Z. Xing and&nbsp;<strong>W. Zhang</strong>&nbsp;(2005) MaxSolver: An efficient exact algorithm for maximum satisﬁability,&nbsp;<em><span style="text-decoration:underline;">Artiﬁcial Intelligence</span></em>, 164(1-2):47-80, 2005.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(244, 176, 131);">AI</span><span style="font-size: 15px;font-family: Arial, sans-serif;">]&nbsp;<strong>W. Zhang</strong>, G. Wang, Z. Xing and L. Wittenberg (2005) Distributed stochastic search and distributed breakout: Properties, comparison and applications to constraint optimization problems in sensor networks,&nbsp;<em><span style="text-decoration:underline;">Artiﬁcial Intelligence</span></em>, 161(1-2):55-87, 2005.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(196, 89, 17);">genom</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">]&nbsp;</span><span style="font-size: 15px;font-family: Arial, sans-serif;">J. Ruan, G. Stormo and&nbsp;<strong>W. Zhang*</strong>, ILM: A web server for predicting RNA secondary structures with pseudoknots,&nbsp;<em><span style="text-decoration:underline;">Nucleic Acids Research</span></em>, 32:W146-9, 2004.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(196, 89, 17);">genom</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">]&nbsp;</span><span style="font-size: 15px;font-family: Arial, sans-serif;">J. Ruan, G. Stormo and&nbsp;<strong>W. Zhang*</strong>,&nbsp;An iterated loop matching approach to the prediction of RNA secondary structures with pseudoknots,&nbsp;<em><span style="text-decoration:underline;">Bioinformatics</span></em>, 20(1):58-66, 2004.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(244, 176, 131);">AI</span><span style="font-size: 15px;font-family: Arial, sans-serif;">]&nbsp;<strong>W. Zhang</strong>&nbsp;(2004) Conﬁguration landscape analysis and backbone guided local search: Part I: satisﬁability and maximum satisﬁability,&nbsp;<em><span style="text-decoration:underline;">Artiﬁcial Intelligence</span></em>, 158(1):1-26, 2004.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(244, 176, 131);">AI</span><span style="font-size: 15px;font-family: Arial, sans-serif;">] A. K. Sen, A. Bagchi and&nbsp;<strong>W. Zhang</strong>&nbsp;(2004) Average-case analysis of best-ﬁrst search in directed acyclic graphs,&nbsp;<em><span style="text-decoration:underline;">Artiﬁcial Intelligence</span></em>, 155(1-2):183-206, 2004.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(244, 176, 131);">AI</span><span style="font-size: 15px;font-family: Arial, sans-serif;">]&nbsp;<strong>W. Zhang</strong>&nbsp;(2004) Phase transitions and backbones of the asymmetric Traveling Salesman Problem,&nbsp;<em><span style="text-decoration:underline;">J. Artiﬁcial Intelligence Research</span></em>, 20:471-97, 2004.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(244, 176, 131);">AI</span><span style="font-size: 15px;font-family: Arial, sans-serif;">]&nbsp;<strong>W. Zhang</strong>&nbsp;(2001) Iterative state-space reduction for ﬂexible computation,&nbsp;<em><span style="text-decoration:underline;">Artiﬁcial Intelligence</span></em>, 126(1-2):109-38, 2001.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(244, 176, 131);">AI</span><span style="font-size: 15px;font-family: Arial, sans-serif;">] M. Tambe and&nbsp;<strong>W. Zhang</strong>&nbsp;(2000) Towards ﬂexible team work in persistent teams: Extended report,&nbsp;<em><span style="text-decoration:underline;">Autonomous Agents and Multi-Agent Systems</span></em>, 3(2):159-83, 2000. (</span><strong><em><span style="text-decoration:underline;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(0, 112, 192);">Best of ICMAS-98</span></span></em></strong><span style="font-size: 15px;font-family: Arial, sans-serif;">).</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(244, 176, 131);">AI</span><span style="font-size: 15px;font-family: Arial, sans-serif;">]&nbsp;<strong>W. Zhang</strong>&nbsp;and R. E. Korf (1996) A study of complexity transitions on the asymmetric Traveling Salesman Problem,&nbsp;<em><span style="text-decoration:underline;">Artiﬁcial Intelligence</span></em>, 81(1-2):223-39, 1996.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(244, 176, 131);">AI</span><span style="font-size: 15px;font-family: Arial, sans-serif;">] J. C. Pemberton and&nbsp;<strong>W. Zhang</strong>&nbsp;(1996) Epsilon-transformation: Exploiting phase transitions to solve combinatorial optimization problems,&nbsp;<em><span style="text-decoration:underline;">Artiﬁcial Intelligence</span></em>, 81(1-2):297-325, 1996.</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(244, 176, 131);">AI</span><span style="font-size: 15px;font-family: Arial, sans-serif;">]&nbsp;<strong>W. Zhang</strong>&nbsp;and R.E.Korf (1995) Performance of linear-space search algorithms,&nbsp;<em><span style="text-decoration:underline;">Artiﬁcial Intelligence</span></em>, 79(2):241-92, 1995. (Impact factor=14.050)</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(244, 176, 131);">AI</span><span style="font-size: 15px;font-family: Arial, sans-serif;">]&nbsp;<strong>W. Zhang</strong>&nbsp;(1989) Representation of assembly and automatic robot planning by Petri net,&nbsp;<em><span style="text-decoration:underline;">IEEE Trans. on Systems, Man and Cybernetics</span></em>, 19(2):418-22, 1989.</span></p></li></ol><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 20px;"><strong><em><span style="font-family: Arial, sans-serif; color: rgb(51, 51, 51);">Selected Peer-reviewed Conference Publication</span></em></strong></span></p><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left; text-indent: 0em;"><span style="font-size: 15px;font-family: Arial, sans-serif;">(Note: IJCAI and AAAI are two leading International AI Confs that adopt double-blind review with acceptance rates being typically around 20%)</span></p><ol style="list-style-type: decimal;" class=" list-paddingleft-2"><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);"><span style="text-wrap: wrap; font-size: 15px; font-family: Arial, sans-serif; color: rgb(51, 51, 51);">[</span><span style="text-wrap: wrap; font-size: 15px; font-family: Arial, sans-serif; color: rgb(68, 114, 196);">ml/dm</span><span style="text-wrap: wrap; font-size: 15px; font-family: Arial, sans-serif; color: rgb(51, 51, 51);">] D. He, J. Zhao, R. Guo, Z. Feng, D. Jin, Y. Huang <span style="color: rgb(51, 51, 51); font-family: Arial, sans-serif; font-size: 15px; text-wrap: wrap; background-color: rgb(255, 255, 255);">, Z. Wang&nbsp;</span>and&nbsp;<strong>W. Zhang</strong>, Contrastive learning meets homophily: Two birds with one stone.&nbsp;<em style="color: rgb(51, 51, 51); font-family: Arial, sans-serif; font-size: 15px; text-wrap: wrap; background-color: rgb(255, 255, 255);"></em><em style="color: rgb(51, 51, 51); font-family: Arial, sans-serif; font-size: 15px; text-wrap: wrap; background-color: rgb(255, 255, 255);">Conf. on Machine Learning</em><span style="color: rgb(51, 51, 51); font-family: Arial, sans-serif; font-size: 15px; text-wrap: wrap; background-color: rgb(255, 255, 255);">&nbsp;(ICML-2023)</span><span style="color: rgb(51, 51, 51); font-family: Arial, sans-serif; font-size: 15px; text-wrap: wrap; background-color: rgb(255, 255, 255);"></span></span></span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(68, 114, 196);">ml/dm</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] D. Jin, R. Wang, M. Ge, H. He, X. Li, W. Lin and&nbsp;<strong>W. Zhang</strong>,&nbsp;RAW-GNN: Random Walk Aggregation based graph neural network,&nbsp;<em>Proc 31-st Intern. Joint Conf. on AI&nbsp;</em>(IJCAI-22).</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(68, 114, 196);">ml/dm</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] D. He, Y. Song, D. Jin, Z. Feng, B. Zhang, Z. Yu and&nbsp;<strong>W. Zhang</strong>,&nbsp;Community-centric graph Convolutional Network for unsupervised community detection,&nbsp;<em>Proc 29-th Intern. Joint Conf. on AI&nbsp;</em>(IJCAI-20).</span></p></li><li><p style="margin: 8px 0px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(244, 176, 131);">AI</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">]&nbsp;X. Zhang, J. Gao, Y. Lv and&nbsp;<strong>W. Zhang</strong>,&nbsp;Early and efficient identification of useless constraint propagation for alldifferent constraints,&nbsp;<em>Proc 29-th Intern. Joint Conf. on AI&nbsp;</em>(IJCAI-20).</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(68, 114, 196);">ml/dm</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] D. Jin, B. Li, P. Jiao, D. He and&nbsp;<strong>W. Zhang</strong>, Network-specific variational Auto-Encoder for embedding of attribute networks,&nbsp;<em>Proc 28-th Intern. Joint Conf. on AI&nbsp;</em>(IJCAI-19).</span></p></li><li><p style="margin: 8px 0px; font-size: 16px; font-family: Calibri, sans-serif; white-space: normal; background: white; text-align: left;"><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">[</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(68, 114, 196);">ml/dm</span><span style="font-size: 15px;font-family: Arial, sans-serif;color: rgb(51, 51, 51);">] D. Jin, Z. Liu, W. Li, D. He and&nbsp;<strong>W. Zhang</strong>, Graph convolutional networks meet Markov Random Fields(<a href="https://www.cse.wustl.edu/%7Ezhang/publications/MRF-aaai18.pdf" style="color: rgb(211, 1, 33);">He, et al.,&nbsp;<em>Proc. AAAI-18</em></a>).</span></span></p><p style="margin: 0in 0in 16px; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;"><span style="font-size: 20px; font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"><strong><em>Applications of network module analysis</em></strong><strong>:</strong></span><span style="font-family: arial, helvetica, sans-serif; font-size: 20px;"><span style="font-size: 20px; font-family: arial, helvetica, sans-serif;"></span><span style="font-size: 20px; font-family: &quot;arial black&quot;, &quot;avant garde&quot;;">&nbsp;</span><span style="font-size: 20px; font-family: Arial, sans-serif;">One of the applications of this line of research is the network-based GWAS, as discussed in&nbsp;</span><span style="font-family: Arial, sans-serif; font-size: 20px;"><a href="https://www.ncbi.nlm.nih.gov/pubmed/25233071" style="color: rgb(211, 1, 33);">(Climer, Templeton and Zhang,&nbsp;<em>PLOS Comp. Biol.</em>, 10(9):e1003766, 2014</a>) and (<a href="https://www.ncbi.nlm.nih.gov/pubmed/25813846" style="color: rgb(211, 1, 33);">Climer, Templeton and Zhang,&nbsp;<em>Nature Comm.</em>, 6:6534, 2015</a>). Another area of application is network-based transcriptome analysis for elucidation of gene regulation, which has been studied in the Zhang lab, as discussed in (<a href="https://www.ncbi.nlm.nih.gov/pubmed/18842138" style="color: rgb(211, 1, 33);">Ray,&nbsp;Ruan&nbsp;and Zhang,&nbsp;<em>Genome Biol.</em>, 9(10):R148, 2008</a>),&nbsp;<a href="https://bmcsystbiol.biomedcentral.com/track/pdf/10.1186/1752-0509-4-8" style="color: rgb(211, 1, 33);">(Ruan, Dean and Zhang,&nbsp;<em>BMC Syst. Biol.</em>, 4:136, 2010</a>) and&nbsp;<a href="https://www.ncbi.nlm.nih.gov/pubmed/18351912" style="color: rgb(211, 1, 33);">(Ruan&nbsp;and Zhang,&nbsp;<em>Physics Review E</em>, 77:016104, 2008</a>).<a class="easyScholarDOI" target="_blank" href="https://sci-hub.se/10.1186/1752-0509-4-8"><svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="22px" height="22px" viewBox="0 0 32 32" enable-background="new 0 0 32 32" xml:space="preserve">  <image id="image0" width="32" height="32" x="0" y="0" href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAABmJLR0QA/wD/AP+gvaeTAAAHdklEQVRYw+1WW2xcVxVd59zn3JnxPDwzHo89fsexYzvO02nSPBQgRMGqVEgFtEiABCriK0L8RHwgJFQJfpCgX6SFQgskqAlRcKAlhJa2zosSJ07i2LXj58Rjz4zHd9535r4OH3YSkiYQARI/WdLRlY7u2Xvdu89aewNP8AT/Z5DHeakmXMs3d25usKRApGBwzrJuc5ZtW7D0Cge9RCtqMp+YXEzGZ8v/UwKR+gaxofdTT6VQ/4mK4NsQqvaFo0GXFPTIkEUOFd20UpmSFkuk04uLySlWTAw5iuPnZ6+9P2Waxn9HYNOuAy2qa+PzXFV0394trU3PbG/x9TT7eb9bgihQEAIwBhimDTVfNkdm0tpbF6dTf7o4OppPTJ11ZYdOT1wdjP9HBNbv/lxvQtn4ze2be/YeOrgx3NdZI4s8BRjAAICxleedIISAEMAwLPP61FL55RNX4mfOD58LGRPHbg8NvJ/JqPajCHAPbnQ//Wx3yr31Wy/079j30te213Y2Vou5oo5c0UDZsHBpLImiZoIQAqfMgxDAshksm4HjKI0EXeKeDfWetMY3XIrRtWs61rmaq7nY3OxM8d8SaO3aUq36dx36woGnP/Pdr/QF/W4HDwDzSyUIPEWVIuDCaBK1fgVhv4KF5RKyRQNT8Rxup4sIemRwlECRBdoR9Urnx3OhyTTX1tvT3bB3Q21mbOyj+YpuPJqAs/PZg90bt77wgxd3NgQ8Dt5mDIQQpLJlKBIPr0uGIvHIlXQ011YhldVwYTQFReZBQOB3S5BFDowBHqdEP4oVxOuxoi9dFqLPP7O789Nbwk41OTc3u5Ap3cnJ3/369Tsjmrtx/9f7u2sjASdv2ytVZmCwbQaeoyiVDUwv5hDyOhBL5mFZDC6Zh5qvwCFx4DmKO5eDEgKRpyCE8gWDBGMFx1NfPfiNSMRV7Gg9dvTnr52J37iPgOnr3LZuTUPnng1RN2P3rphtAy4HD5eDR7FsQhQ4ZIo6qhQB7fUetEaq7v1OSsAAUEqQUDUMTS+DEsBmhLcYdQUCgfbN+1/0Sbd/5w14ml4fuMaP8wDQtrbbkxfD27Z1RcPVbpm3VwlQQgAKREMuMAZ4XRz29tZiVQgAAIEjYFgp1R0kMxp+fHIUN2czIIRAESk2rfGDUso73b5gS9S3u784JIQa+sd4n9frbO/pW3e57O9oi/rdIABhwFKujLFYFoQSVLsleJ0iFImHwFNwlNz1AdNmqOgWskUdC6qG69Mq3rocx7UpFZbNIPAUX9zdhF1dNSvVSQ/zkjblCzqxZi1JKHwo4AnvaDFad9c7wl31Th4gYITA65IQ8Mh4d3gRl8eXkcqvuKzMc5BEHhwlsC0bmm4hV9ahFnRkCwZKFROEAG6HgLa6Kjy3swGf3dEIhyzATA5Bu/Qd09IyhmYiu2RIizwjIoMcMvfXXEPr4t9RMXvBhbaDVjWjM+pFR4MPamFFajfnMpiM5zG/VIRa1KHpFizbBqUE9QEnuqMiwn4HWiJudDX6sDbqhc9JYOdmUBk7gcrIK6aZmSmn8hgbTvr/eGqaDROPx6N8sv/z6yvBvsOH9sl7tyuDspm8DMJJoP51oNXrwXnawbnrQCQ/GOeAyQToFoFhMdgMIATgKSByNgQYoFYOdiEOM30D5sIgrIVBWLlZs1yx1UQeIxfjwWOvXao6OXRjKkMAoKOzy11pee7whk19X/rZt3cFFSPGm/PvwLp9Flb6OlBZBqgIIvlAZD+I5AcRqwDeAUJ4MGYBpgam58DKS7C1FFBOgVXygM3AALNgcAu38sF3/zxd98aRgYlBNZOz7+sFG/d9eafq2fa9Vw/3b93TWyfbAGDqsEtx2Ooo7PQV2Ms3YWcnwbQEoGfBTA2wjRXts9VoBADlQTgHiOSF5YiYKdasjmQip0/8ZeInR48P3Pxn87vrA7NX3r4o9gT/cOT3wzW9rdUtHqfIM8qDuptAq5qAhgOAbYKZRbBKBqioYBUVTM8BZgnMNgFCQDgJENygsh+m4McrZxbLA+cmh0OlK7868UDy+6xYKxXtgGJPj2ccIU5y1fd1hJ08RykYWxU9W00gg0heECUM6mkG9XWAVveAC/SCC6wH9XeB87aBuurw5oV04YdHhyaU4q03Lpx+dUDX9X/djDKp+WK1S5z5cA5BTlRqetuCTlHgKGMPHlslxD6+CADTtMzj701kX/rFBzfF3MTruZFTv0kmFh86oXysHecSU8tehU6cu1VxxjOWp7XO6/C5ZcpxlILhkaCEgIGZiXRBf/nk1cSPfv3eIEtd/eny8PE347dj+qPOcQ/bzCWmln1C8eq1mFY4O5zi1bxB3Q6ROh08EXgOhMAG7i2tbOhTcTV//K/jqe//8oMbb79z8ZSy/OGRyfO/PaeVio8cRu5TwcMgiBJq1u5sLbvatlcF6je1N0Ua2xuD1SGfwimyIFQqppHKlqxbsXRmfHZxPp2I35BL0+cLc38bUZcSJh4DjzUVA4AvVC+J3mjQEjx+RiWFEULAGCN2pcQZOdXMxZeWF2e0x433BE9wB/8Aa3ZiVq1gDAMAAAAldEVYdGRhdGU6Y3JlYXRlADIwMjItMTItMDFUMTQ6NDQ6MDIrMDE6MDBW0C0AAAAAJXRFWHRkYXRlOm1vZGlmeQAyMDIyLTEyLTAxVDE0OjQ0OjAyKzAxOjAwJ42VvAAAAABJRU5ErkJggg=="></image></svg></a></span></span></span><span style="font-size: 18px;"><span style="font-size: 16px;"><strong style="font-size: 18px;"><span style="font-size: 19px; font-family: Arial, sans-serif;"><br></span></strong></span></span></p><p style="margin: 0in 0in 16px; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-size: 18px;"><span style="font-size: 20px; font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"><strong><span style="font-size: 24px;">Research interests - Heuristic search, planning and optimization</span></strong></span><span style="font-size: 20px;"><span style="font-size: 20px; font-family: arial, helvetica, sans-serif;"><strong><span style="font-size: 24px;"></span></strong></span><span style="font-family: Arial, sans-serif; font-size: 18px;"><br></span></span></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: Arial, sans-serif; font-size: 18px;">The Zhang has a long history of research on heuristic search, planning and optimization. The work has made significant contributions to the understanding of the nature and complexity of many search techniques, such as best-first search and depth-first branch-and-bound, and combinatorial optimization problems, such as the Traveling Salesman Problem and Constraint Satisfaction.</span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;; font-size: 20px;"><strong><em>Asymptotic optimality of linear-space search algorithms:</em></strong></span><span style="font-size: 18px;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"><strong><em> </em></strong></span><span style="font-family: Arial, sans-serif;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">Heuristic search is one of the cornerstone of AI. The best-first search (BFS) algorithm, with the well-known A* algorithm as a special case, is time optimal. However, it requires space that is exponential in the search depth of the underlying search tree, which is impractical for large combinatorial optimization problems, such as the Traveling Salesman Problem (TSP). In contrast, a linear-space search (LSS) algorithm, with depth-first search and depth-first branch-and-bound (DFBnB) as special cases, runs linearly in search depth with a penalty of exploring more nodes in the search space. Due to their limited space requirement, LSS algorithms, including&nbsp;DFBnB, iterative deepening A* (IDA*), and recursive best-first search (RBFS), are the most popular techniques for large and difficult search problems in practice. Understanding the expected complexity of LSS algorithms was an important and challenging&nbsp;problem.Based&nbsp;on a random search tree model proposed by Judea Pearl and Richard Karp,&nbsp;<strong>our research established the&nbsp;<em><span style="font-family: arial, helvetica, sans-serif; text-decoration: underline;">asymptotic optimality</span>&nbsp;</em>of LSS methods</strong>, showing that their expected complexity is a constant factor of that of BFS (</span><a href="https://www.cse.wustl.edu/%7Ezhang/publications/bnbaij.pdf" style="color: rgb(211, 1, 33); text-decoration: underline; font-family: arial, helvetica, sans-serif; font-size: 18px;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">Zhand and Korf,&nbsp;<em>Artificial Intelligence</em>, 79(2):241-92, 1995</span></a><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">;&nbsp;</span><a href="https://www.springer.com/us/book/9780387988320" style="color: rgb(211, 1, 33); text-decoration: underline; font-family: arial, helvetica, sans-serif; font-size: 18px;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">Zhang,&nbsp;<em>State-Space Search</em>, Springer, 1999</span></a><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">). Moreover, LSS algorithms may run faster than BFS on large problems, making them the algorithms for real applications.</span></span></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">A directly and early application of our complexity results is the explanation to anomaly of the fixed horizon search, which was observed initially by Richard&nbsp;Korf&nbsp;on sliding-tile puzzles using&nbsp;DFBnB, as illustrated below. The anomaly states that for a fixed search depth,&nbsp;DFBnB&nbsp;can search the trees with larger branching factors faster. In fact, the anomaly persists regardless which search method, e.g., ID (or IDA*) or RBFS, shown below.</span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left; text-indent: 0px;"><span style="font-family: Arial, sans-serif; font-size: 18px; text-indent: 0px;"><img title="11.png" style="border: none; width: 365px; height: 344.722px;" src="/picture/0/dd7eea7dbc684d22aa404423aca5e170.png" width="365" height="344"><img title="12.png" style="border: none; width: 661px; height: 331.86px;" src="/picture/0/24b0a4109cf841159f9678ca1c488c97.png" width="661" height="331"></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: Arial, sans-serif; font-size: 18px;">The insight gained by the complexity analysis on this search anomaly helped predict the existence of a phase transition in heuristic search problems, discussed below.</span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;; font-size: 20px;"><strong><em>Phase transitions and structures of combinatorial search: </em></strong></span><span style="font-size: 18px;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"></span><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">Our study also revealed that the expected complexity of BFS and LLS algorithms exhibits a sharp and abrupt transition from polynomial to exponential time as some critical problem features (e.g., the precision of heuristic function) shift (<a href="https://www.cse.wustl.edu/%7Ezhang/publications/bnbaij.pdf" style="color: rgb(211, 1, 33);">Zhand and Korf,&nbsp;<em>Artificial Intelligence</em>, 79(2):241-92, 1995</a>;&nbsp;<a href="https://www.cse.wustl.edu/%7Ezhang/publications/tspaij.ps" style="color: rgb(211, 1, 33);">Zhang and&nbsp;Korf,&nbsp;<em>Artificial Intelligence</em>, 81(1-2):223-39, 1996</a>;&nbsp;<a href="https://www.springer.com/us/book/9780387988320" style="color: rgb(211, 1, 33);">Zhang,&nbsp;<em>State-Space Search</em>, Springer, 1999</a>). This is similar to phase transitions of spin glasses studied in physics. A simple example of a phase transition is water changing from liquid to solid ice when the temperature drops below the freezing point. We identified that the accuracy of heuristic information is the controlling parameter of phase transitions. On the&nbsp;Judea&amp;Karp&nbsp;incremental random search tree model, the phase transition can be summarized by the following figure.</span></span></p><p style="margin: 0in 0in 0px; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left; text-indent: 0px;"><span style="font-family: Arial, sans-serif; color: rgb(0, 0, 153); font-size: 18px; text-indent: 0px;"><img title="13.png" style="border: none; width: 402px; height: 385.048px;" src="/picture/0/2eac6d39e18b4de4b02f846110bb9b8e.png" width="402" height="385"></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: Arial, sans-serif; font-size: 18px;">The control parameter of this phase transition is the product of the mean branching factor of the search tree and the probability if an edge cost has cost 0.</span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: Arial, sans-serif; font-size: 18px;">This analysis was extended to reveal phase transitions in combinatorial optimization problems, such as the asymmetric TSP (ATSP) (<a href="https://www.sciencedirect.com/science/article/pii/0004370295000542" style="color: rgb(211, 1, 33);">Zhang and&nbsp;Korf,&nbsp;<em>Artificial Intelligence</em>, 81(1-2):223-39, 1996</a>) and Boolean Satisfiability (<a href="https://www.sciencedirect.com/science/article/pii/S0004370204000542" style="color: rgb(211, 1, 33);">Zhang,&nbsp;<em>Artificial Intelligence</em>, 158:1-26, 2004</a>). For the ATSP, the control parameter is the precision of the distances between cities, which is equivalent to the number of digits used to represent distances. Intuitively, when the number of digits used is small, many distances may have a high chance to have the same value, and consequently many complete tours (visiting each and every city exactly once) may have a high chance to have the same cost as well. This also means that a large number of optimal TSP tours exist; and finding one optimal TSP tour should be relatively easy. In contrast, when no two distances between cites are the same, the number of optimal TSP tour is small, so that finding an optimal solution is difficult. This phase transition is illustrated in the figure below, where the Assignment Problem (AP) is the main computation step (heuristic) for solving the ATSP.</span></p><p style="margin: 0in 0in 0px; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left; text-indent: 0px;"><span style="font-family: Arial, sans-serif; font-size: 18px; text-indent: 0px;"><img title="14.png" style="border: none; width: 367.55px; height: 300px;" src="/picture/0/d808eb0cb7c74017aae8fd95733deb5a.png" width="367" height="300"></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: Arial, sans-serif; font-size: 18px;">Interestingly, there also exist phase transitions in maximum Boolean satisfiability (max SAT), which is the optimization version of the SAT where the objective is to find a variable assignment that minimize the number of violated clauses (<a href="https://www.sciencedirect.com/science/article/pii/S0004370204000542" style="color: rgb(211, 1, 33);">Zhang,&nbsp;<em>Artificial Intelligence</em>, 158:1-26, 2004</a>).</span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-size: 18px;"><strong><span style="font-family: Arial, sans-serif;">Note that the phase transitions that we studied in optimization problems, such as finding the minimal TSP tour and the variable assignment to minimize the number of unsatisfied constraints, are fundamentally different from the phase transitions for decision problems, such as the decision version of the TSP and the SAT. For optimization problems, the phase transitions appear as an easy-to-difficult transition, whereas for decision problems, the phase transitions exhibit an easy-to-difficult-then-to-easy pattern.</span></strong><strong><span style="font-family: Arial, sans-serif;"></span></strong></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: Arial, sans-serif; font-size: 18px;">The computational complexity and phase transition are also related to the backbone of a problem, which is a set of variables that have the same values among all solutions (<a href="https://dl.acm.org/citation.cfm?id=726965" style="color: rgb(211, 1, 33);">Zhang,&nbsp;<em>Proc. Intern. Conf. on Principles and Practice of Constraint programming</em>, CP-2001</a>;&nbsp;<a href="https://arxiv.org/pdf/1107.0055.pdf" style="color: rgb(211, 1, 33);">Zhang,&nbsp;<em>JAIR</em>, 20:471-97, 2004</a>;&nbsp;<a href="https://www.sciencedirect.com/science/article/pii/S0004370204000542" style="color: rgb(211, 1, 33);">Zhang,&nbsp;<em>Artificial Intelligence</em>, 158:1-26, 2004</a>).</span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;; font-size: 20px;"><strong><em>Problem solving by exploiting phase transitions and backbones: </em></strong></span><span style="font-size: 18px;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"></span><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">The study of phase transitions and problem structures such as backbones has changed the way we characterize the complexity of combinatorial problems, beyond the notion of worst-case complexity, and the way we solve difficult problems.&nbsp; In particular, phase transitions can not only be used to characterize problems, but also be exploited in problem solving. We have done a&nbsp;<strong>pioneering work</strong>&nbsp;on developing new efficient algorithms for finding optimal and approximate solutions by exploiting phase transitions (<a href="https://ac.els-cdn.com/0004370295000577/1-s2.0-0004370295000577-main.pdf?_tid=8adf368a-3539-4297-975c-a6d046a35b9f&amp;acdnat=1539208483_3dee19b9b4d695046e4d089ecb64578f" style="color: rgb(211, 1, 33); text-decoration: underline; font-family: arial, helvetica, sans-serif; font-size: 16px;">Pemberton and Zhang,&nbsp;<em>Artificial Intelligence</em>, 81:297-325, 1996</a>;&nbsp;<a href="https://www.sciencedirect.com/science/article/pii/S0004370204000542" style="color: rgb(211, 1, 33); text-decoration: underline; font-family: arial, helvetica, sans-serif; font-size: 16px;">Zhang,&nbsp;<em>Artificial Intelligence</em>, 158:1-26, 2004</a>). One of the ideas is to transform a state space that is difficult to search into one that is easy to explore. Another idea is to estimate and utilize backbone structures to guide a search. These techniques are characterized as&nbsp;<strong>backbone guided search</strong>&nbsp;(<a href="https://www.sciencedirect.com/science/article/pii/S0004370204000542" style="color: rgb(211, 1, 33); text-decoration: underline; font-family: arial, helvetica, sans-serif; font-size: 18px;">Z<span style="font-family: arial, helvetica, sans-serif; color: rgb(211, 1, 33); text-decoration: underline;">hang,&nbsp;<em>Artificial Intelligence</em>, 158:1-26, 2004</span></a>;&nbsp;<a href="https://www.cse.wustl.edu/%7Ezhang/publications/bgtsp.pdf" style="color: rgb(211, 1, 33); text-decoration: underline; font-family: arial, helvetica, sans-serif; font-size: 16px;">Zhang and Looks,&nbsp;<em>Proc. IJCAI-2005</em></a>).</span></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: arial, helvetica, sans-serif; font-size: 20px;"><strong><em><span style="font-family: Arial, sans-serif;">T</span></em></strong></span><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"><span style="font-size: 20px;"><strong><em>he Zhang Algorithm for the ATSP:</em></strong></span><span style="font-size: 18px;"><strong><em> </em></strong></span></span><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;"><strong><em><span style="font-family: Arial, sans-serif;"></span></em></strong><span style="font-family: Arial, sans-serif;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">The asymmetric TSP (ATSP) is an important NP-hard problem with many real applications in the areas such as scheduling. The research largely focuses on efficient approximation algorithms. The first approximation method is the&nbsp;Kanallakis-Papadimitiou&nbsp;local search algorithm, named after its inventors, two eminent scientists in computer science and operations research. The second approach is represented by my truncated&nbsp;DFBnB&nbsp;algorithm (</span><a href="https://www.cse.wustl.edu/%7Ezhang/publications/atsp-aaai93-symp.ps" style="color: rgb(211, 1, 33); text-decoration: underline; font-family: arial, helvetica, sans-serif; font-size: 18px;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">Zhang,&nbsp;<em>Proc. AAAI 1993 Spring&nbsp;Symp. on AI and NP-Hard Problems</em></span></a><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">;&nbsp;</span><a href="https://www.springer.com/us/book/9780387988320" style="color: rgb(211, 1, 33); text-decoration: underline; font-family: arial, helvetica, sans-serif; font-size: 18px;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">Zhang,&nbsp;<em>State-Space Search</em>, Springer, 1999</span></a><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">), which has been referred to as&nbsp;<strong>the Zhang algorithm</strong>&nbsp;(</span><a href="http://dimacs.rutgers.edu/archive/Challenges/TSP/papers/stspchap.pdf" style="color: rgb(211, 1, 33); text-decoration: underline; font-family: arial, helvetica, sans-serif; font-size: 18px;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">J<span style="font-size: 18px; font-family: Arial, sans-serif; color: rgb(211, 1, 33);">ohnson, et al.in&nbsp;<em>The Traveling Salesman Problem and its Variations</em>, pp.445-88, 2002</span></span></a><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">).</span></span></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-size: 18px;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"><strong><em><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;; font-size: 20px;">Long distance mutual exclusion and propositional planning: </span></em></strong></span><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">A dominating technique in planning is&nbsp;GraphPlan&nbsp;with mutual exclusions (mutex) to represent constraints among actions and events at the same planning horizon (depth). In our work, we introduced long distance mutual exclusion (londex) to capture constraints across multiple horizons, developed an efficient algorithm to efficiently compute&nbsp;londex, and designed and implemented our new&nbsp;MaxPlan&nbsp;planner using&nbsp;londex&nbsp;(<a href="https://www.cse.wustl.edu/%7Ezhang/publications/MaxPlan-final.pdf" style="color: rgb(211, 1, 33);">Xing, Chen and Zhang,&nbsp;<em>Proc. ICAPS-06</em></a>;&nbsp;<a href="https://www.cse.wustl.edu/%7Ezhang/publications/londexaij.pdf" style="color: rgb(211, 1, 33);">Chen, Huang, Xing and Zhang,&nbsp;<em>Artificial Intelligence</em>, 173:365-91, 2009</a>). The overall work provided a new paradigm for planning. Our&nbsp;MaxPlan&nbsp;system won&nbsp;<strong>the First Place Award of the Fifth International Planning Competition in 2006</strong>&nbsp;(<a href="https://www.cse.wustl.edu/%7Ezhang/publications/MaxPlan-final.pdf" style="color: rgb(211, 1, 33);">Xing, Chen and Zhang,&nbsp;<em>Proc. ICAPS-06</em></a>).</span></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;; font-size: 20px;"><strong><em>Transition based encoding for planning:</em></strong></span><span style="font-size: 18px;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"><strong><em> </em></strong></span><span style="font-family: Arial, sans-serif;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">The technique of planning as Boolean satisfiability usually uses encodings in logic statements. We introduced a novel SAT encoding scheme (SASE) based on the SAS+ formalism (</span><a href="https://www.cse.wustl.edu/%7Ezhang/publications/SASE-Encoding-aaai10.pdf" style="color: rgb(211, 1, 33); text-decoration: underline; font-family: arial, helvetica, sans-serif; font-size: 18px;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">Huang, Chen and Zhang,&nbsp;<em>Proc. AAAI-10</em></span></a><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">;&nbsp;</span><a href="https://jair.org/index.php/jair/article/view/10751" style="color: rgb(211, 1, 33); text-decoration: underline; font-family: arial, helvetica, sans-serif; font-size: 18px;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">Huang, Chen and Zhang,&nbsp;<em>JAIR</em>, 43:293-328, 2012</span></a><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">). The new scheme exploits the structural information so that the encoding is both compact and efficient for planning. We proved the correctness of the new encoding and analyzed the rationale behind its improvement in planning performance. Our extensive experimental results demonstrated significant improvements of SASE over the state-of-the-art methods in time and space. This work received&nbsp;<strong>the Outstanding Paper Award</strong>&nbsp;of the 24-th AAAI Conference on Artificial Intelligence Conference in 2010 (AAA-10) (</span><a href="https://www.cse.wustl.edu/%7Ezhang/publications/SASE-Encoding-aaai10.pdf" style="color: rgb(211, 1, 33); text-decoration: underline; font-family: arial, helvetica, sans-serif; font-size: 18px;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">Huang, Chen and Zhang,&nbsp;<em>Proc. AAAI-10</em></span></a><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">), chosen from a double blind review process of 894 submissions. Note that AAAI is one of the two leading international AI conferences.</span></span></span></p><p><br></p><meta name="ContentEnd"><!--ZJEG_RSS.content.end--><!--<$[信息内容]>end-->
	</div>
	    
	<div class="share">
	  
	  <a class="close1" href="javascript:window.opener=null;window.open('','_self');window.close();">close</a>
		<a class="print" href="javascript:window.print()">print</a>
	</div></div>
 
                </div>
                
                  

                </div>      
              </div>
      </div>
      
      </div>
    </div>
  </div>
</section>
<footer id="footer">
<div class="container">
  <div class="row">
    <div class="col-md-12">
      <ul>
        <li>Contact: The Hong Kong Polytechnic University Lee Shau Kee Building, BlockY 922,
          Yuk Choi Rd, Hung Hom, Hong Kong
           </li>

       
      </ul>
    </div>
  </div>
</div>
</footer>
<script type="text/javascript" src="/site/images/6964/common.js"></script>


</body>
</html>
